src/resources/interfaces.ts

```diff
@@ -24,7 +24,7 @@ interface Property {
   name: string;
   comment: string;
   type: string;
-  access: string;
+  access?: string;
```

Comment:
Discussed offline, access right now tells us if a parameter is read-only v.s. create-only. In reality, we expect all the params today to be read-only so I am going to take out the access field from the interface until we decide otherwise.